share/completions/fd.fish

```diff
@@ -0,0 +1,34 @@
+#fd 7.4.0
```

Comment:
fd ships its own completions, please submit any enhancements to them upstream.

Reply:
You are right, I forgot about `fish_update_completions` on one host.